src/Tests/Microsoft.NET.Build.Tests/GivenFrameworkReferences.cs

```diff
@@ -336,10 +336,10 @@ public void TargetingPackDownloadCanBeDisabled()
         }
 
         [Theory]
-        [InlineData("Major", "netcoreapp3.0", true)]
+        [InlineData("Major", ToolsetInfo.CurrentTargetFramework, true)]
         [InlineData("Major", "netcoreapp2.0", true)]
-        [InlineData("latestMinor", "netcoreapp3.0", true)]
-        [InlineData("Invalid", "netcoreapp3.0", false)]
+        [InlineData("latestMinor", ToolsetInfo.CurrentTargetFramework, true)]
+        [InlineData("Invalid", ToolsetInfo.CurrentTargetFramework, false)]
```

Comment:
Reading this test, I think it's trying to rollforward to the latest minor. Since 5.0 and 6.0 don't have minors, it probably makes sense to keep this on 3.0.